javascripts/discourse/initializers/init-header-user-menu.js

```diff
@@ -0,0 +1,12 @@
+import { apiInitializer } from "discourse/lib/api";
```

Comment:
IMO you could combine both these initializers into one, they are tiny and there doesn't seem to be much benefit to splitting them